pkg/command/service/load.go

```diff
@@ -0,0 +1,423 @@
+			if err == nil {
+				// print result(load test tool output, replicas result, pods result)
+				if inputs.Verbose {
+					fmt.Printf("\n[Verbose] Namespace %s, Service %s:\n", loadResult.ServiceNamespace, loadResult.ServiceName)
+					fmt.Printf("\n[Verbose] %s output:\n%s\n", inputs.LoadTool, loadToolOutput)
+					fmt.Printf("[Verbose] Deployment replicas changed from 0 to %d:\n", len(loadResult.ReplicaResults))
+					fmt.Printf("replicas\tready_duration(seconds)\n")
+					for i := 0; i < len(loadResult.ReplicaResults); i++ {
+						fmt.Printf("%13d\t%23.3f\n", i, loadResult.ReplicaResults[i].ReplicaReadyDuration)
+					}
+					fmt.Printf("\n[Verbose] Pods changed from 0 to %d:\n", len(loadResult.PodResults))
+					fmt.Printf("pods\tready_duration(seconds)\n")
+					for i := 0; i < len(loadResult.PodResults); i++ {
+						fmt.Printf("%9d\t%23.1f\n", i, loadResult.PodResults[i].PodReadyDuration)
+					}
+					fmt.Printf("\n---------------------------------------------------------------------------------\n")
+				}
+				m.Lock()
+				result.Measurment = append(result.Measurment, loadResult)
+				m.Unlock()
+			} else {
+				fmt.Printf("result of load is error: %s", err)
+			}
+		}(i, &m)
+	}
+	wg.Wait()
+
+	return result, nil
+}
+
+func runLoadFromZero(ctx context.Context, params *pkg.PerfParams, inputs pkg.LoadArgs, namespace string, svc *servingv1.Service) (
+	string, pkg.LoadFromZeroResult, error) {
+	selector := labels.SelectorFromSet(labels.Set{
+		serving.ServiceLabelKey: svc.Name,
+	})
+	var loadOutput string
+	var loadResult pkg.LoadFromZeroResult
+	var replicaResults []pkg.LoadReplicaResult
+	var podResults []pkg.LoadPodResult
+	loadResult.ServiceName = svc.Name
+	loadResult.ServiceNamespace = namespace
+
+	watcher, err := params.ClientSet.AppsV1().Deployments(namespace).Watch(
+		context.Background(), metav1.ListOptions{LabelSelector: selector.String()})
+	if err != nil {
+		m := fmt.Sprintf("unable to watch the deployment for the service: %v", err)
+		log.Println(m)
+		return "", loadResult, errors.New(m)
+	}
+	defer watcher.Stop()
+
+	rdch := watcher.ResultChan() // replica duration channel
+	pdch := make(chan struct{})  // pod duration channel
+	errch := make(chan error)
+
+	endpoint, err := resolveEndpoint(ctx, params, inputs.ResolvableDomain, svc)
+	if err != nil {
+		return "", loadResult, fmt.Errorf("failed to get the cluster endpoint: %w", err)
+	}
+
+	// Prepare for load test
+	cmd, wrkLua, err := loadCmdBuilder(inputs, endpoint, namespace, svc)
+	if err != nil {
+		return "", loadResult, err
+	}
+
+	defer func() {
+		// Delete wrk lua script
+		if strings.EqualFold(inputs.LoadTool, "wrk") {
+			_, fileError := os.Stat(wrkLua)
+			if fileError == nil {
+				removeError := os.Remove(wrkLua)
+				if removeError != nil {
+					log.Printf("remove %s error : %s", wrkLua, removeError)
+				}
+			}
+		}
+	}()
+
+	loadStart := time.Now()
+	m := fmt.Sprintf("Namespace %s, Service %s, load start", namespace, svc.Name)
+	log.Println(m)
```

Comment:
Why not merge the upper two lines into one line as `log.Printf("Namespace %s, Service %s, load start\n", namespace, svc.Name)` ?

Reply:
Thank you, addressed.